### PR TITLE
V5 update

### PR DESCRIPTION
**Pull request by alexhemard**
## PR summary
- Added additional Connection String types (MySQL, MongoDB EE, Datastax Enterprise)

**Fixes:** 
- Point In Time Restore Data response parsing

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Does this PR introduce a breaking change?    
- [x] Yes
- [ ] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 [![CLA assistant](https://cla-assistant.io/readme/badge/ibm/cloud-databases-go-sdk)](https://cla-assistant.io/ibm/cloud-databases-go-sdk)
 
-# IBM Cloud Databases Go SDK 0.1.0
+# IBM Cloud Databases Go SDK 0.2.0
```
Comment (padamstx) [inline README.md:9]: This should not be done manually.  It will be changed by semantic-release when the PR is merged into main.
```suggestion
# IBM Cloud Databases Go SDK 0.1.0
```

File: README.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ Service Name | Package name
 * Go version 1.14 or above.
 
 ## Installation
-The current version of this SDK: 0.1.0
+The current version of this SDK: 0.2.0
```
Comment (padamstx) [inline README.md:56]: Same here... revert all changes to README.md

File: clouddatabasesv5/cloud_databases_v5.go [hunk 3]
```diff
@@ -15,7 +15,7 @@
  */
 
 /*
- * IBM OpenAPI SDK Code Generator Version: 99-SNAPSHOT-f9bb1b47-20210329-151039
+ * IBM OpenAPI SDK Code Generator Version: 99-SNAPSHOT-05af0f12-20220210-122824
```
Comment (padamstx) [inline clouddatabasesv5/cloud_databases_v5.go]: Technically, you should be using an official version of the SDK generator, rather than a locally-built version.

File: common/version.go [hunk 4]
```diff
@@ -17,4 +17,4 @@
 package common
 
 // Version of the SDK
-const Version = "0.1.0"
+const Version = "0.2.0"
```
Comment (padamstx) [inline common/version.go:20]: Revert this change as it will be done automatically by semantic-release when the PR is merged.

**Review by padamstx — CHANGES_REQUESTED** [review]
Need to revert all release-related changes.
Also should probably update go.mod to use the latest go core (latest is now 5.9.2).

**Review by padamstx — APPROVED** [review]
Changes look good.
A new build for this PR did not get triggered for your latest push for some reason.  I'll re-start the build.
Also, could you provide screenshots showing that the integration tests and examples all run clean?

**Comment by padamstx** [comment]
The restarted PR build was clean, but for some reason the status is not reflected here in the PR:
https://app.travis-ci.com/github/IBM/cloud-databases-go-sdk/builds/246512721

**Comment by alexhemard** [comment]
Passing integration, example tests

```sh
(⎈ |:) cloud-databases-go-sdk (v5-update) $ make test-int
go test `go list ./...` -tags=integration
ok  	github.com/IBM/cloud-databases-go-sdk/clouddatabasesv5	196.379s
ok  	github.com/IBM/cloud-databases-go-sdk/common	(cached)
```
```sh
(⎈ |:) cloud-databases-go-sdk (v5-update) $ go test `go list ./...` -tags=examples
ok  	github.com/IBM/cloud-databases-go-sdk/clouddatabasesv5	204.069s
ok  	github.com/IBM/cloud-databases-go-sdk/common	(cached)
```
